app/assets/stylesheets/application.css.scss

```diff
@@ -51,3 +51,33 @@ path {
   stroke: #fff;
   fill-rule: evenodd;
 }
+
+.d3-tip {
+ line-height: 1;
+ font-weight: bold;
+ padding: 12px;
+ background: rgba(0, 0, 0, 0.8);
+ color: #fff;
+ font-color: white;
+ border-radius: 2px;
+}
+
+/* Creates a small triangle extender for the tooltip */
+.d3-tip:after {
+ box-sizing: border-box;
+ display: inline;
```

Comment:
Line should be indented 2 spaces, but was indented 1 spaces<br>Properties should be ordered box-sizing, color, content, display, font-size, line-height, position, text-align, width